### PR TITLE
docs_low_level: ofBaseRenderer.h (ofBaseGLRenderer)

### PR DESCRIPTION
**Pull request by brannondorsey**
Inline documentation for ofBaseGLRenderer.

### CODE REVIEW

File: libs/openFrameworks/types/ofBaseTypes.h [hunk 1]
```diff
@@ -1786,67 +1786,371 @@ class ofBaseRenderer{
...
+	/// \param drawMode OpenGL primitive draw mode to use when drawing the vbo's
+	/// vertices with this renderer.
+	/// \param amt The number of elements to be rendered.
+	/// \param offsetelements The number of elements (indices) offset from the
+	/// first element to start drawing from.
+	/// \sa ofGetGLPrimitiveMode()
+	/// \sa glDrawElements()
+	/// \sa http://www.opengl-tutorial.org/intermediate-tutorials/tutorial-9-vbo-indexing/
+	/// \sa https://www.opengl.org/sdk/docs/man/html/glDrawElements.xhtml
 	virtual void drawElements(const ofVbo & vbo, GLuint drawMode, int amt, int offsetelements) const=0;
```
Comment (brannondorsey) [inline libs/openFrameworks/types/ofBaseTypes.h]: I had significant trouble understanding how to describe the differences between regular drawing, drawing elements, drawing instanced, and drawing elements instanced, so for these purposes I erred on the side of being vague about their uses so as not to provide misinformation. Where appropriate I linked to the openGL references to each function. Perhaps folks that are more well-versed in these forms of drawing can provide better documentation for these functions.

**Comment by bakercp** [comment]
Hi @arturoc if I get this merge-ready can we merge it (documentation would be nice for a course I'm teaching).

**Comment by arturoc** [comment]
same as all the other docs related they have conflicts and can't be merged anymore

**Comment by bakercp** [comment]
@arturoc @brannondorsey Can you fix? Would like to get these docs merged.

**Comment by bakercp** [comment]
Closed in favor of #6386

**Pull request closed without merging**